### DISCUSSION TITLE
Play game button / Initial Menu

### DISCUSSION THREAD

**Discussion by vfx1b** [excaliburjs/Excalibur#2467] [Q&A]
Hi, I'm building a widget in a transparent electron window and was wondering if its possible to skip the initial menu/loader ? I couldn't find anything in the docs, issues and by googling.

**Comment by eonarheim** [comment]
Hi @vfx1b! Thanks for opening a discussion! 

I'll spend some time today to update the documentation around the possible engine options/customizing the loader.

I think you might be able to use the engine option `suppressPlayButton`, the caveat is that the browser might suppress web audio (but not sure if electron is different and doesn't have the same user requirements as a browser)

```typescript
const game = new ex.Engine({
  suppressPlayButton: true
});
```

As for the loading bar, you could build a custom loader with a transparent background or whatever look and feel you want. A few folks have had some success and some examples in this thread https://github.com/excaliburjs/Excalibur/discussions/2464

Let me know if you have any questions!

**Reply by vfx1b** [reply]
Thank you so much! I'll try this out and report back.

**Comment by mattjennings** [comment]
You can create a custom loader like this to make the loader draw nothing and automatically start:

```ts
class Loader extends ex.Loader {
  showPlayButton() {
    return Promise.resolve();
  }

  draw() {}
}

const loader = new Loader()
```

I typically use something like this during development so I dont have to click play after making changes. However @eonarheim makes a good point that the play button allows audio to be enabled once clicked, so that is something to consider if you use audio.
